### ISSUE TITLE
Improve dark mode contrast

### ISSUE THREAD

**Issue by johnareid54** [dart-lang/dart-pad#1628]
Not sure when but DartPad now seems to only allow dark mode, whereas before it had light mode.

This is making using DartPad very difficult as it is very difficult to see the code against a black background.

Bring back light mode or a least an option to change display themes.

**Comment by RedBrogdon** [comment]
Thanks for filing an issue. Are you referring to the full-screen UI, or the embedded layouts for use in iframes? If it's in the latter, they do support theming. For a demo, see https://dartpad.dev/example/embed.html.

For the fulls-screen UI, there's just the one theme. The font colors are intended to be pretty bright on the dark background, though:

![image](https://user-images.githubusercontent.com/969662/94108437-27cfd180-fdf4-11ea-93fb-29e924e78cac.png)

Are you seeing anything different? If so, can you provide browser details, and let us know if there's any errors in the page load or JavaScript console?

**Comment by johnareid54** [comment]
HI @RedBrogdon 
I have been using the full-screen version at https://dartpad.dev/.

Since it has moved to the dark theme I have stopped using it as it is just too hard to see the detail.
It's a real pity as dartpad was a quick and easy means of trying code out.

**Comment by bradcypert** [comment]
@RedBrogdon Scanning through a couple of recent issues makes me wonder if a small "Settings" menu would be helpful.
I feel like that would help with this issue, #1640, and #1598.

Maybe something similar to how Codepen does settings? We could also persist these into local storage so they remain the default settings across tabs/instances.

**Comment by theacodes** [comment]
I am +1ing this issue and marking it as an accessibility issue. The overall contrast ratio for the dark theme is way too low and it's very difficult to see.

**Comment by theacodes** [comment]
Changing the background color to `#0e161f` and the default foreground "white" to `#fff` alone makes a huge difference.

![image](https://user-images.githubusercontent.com/250995/108121475-de1ca780-7070-11eb-8d49-3366fd6e95fa.png)

Switching the greys to white and bumping the lightness up the blue, green, and red colors further improves legibility:

![image](https://user-images.githubusercontent.com/250995/108121941-89c5f780-7071-11eb-96b9-c560f30602a5.png)

**Comment by russeg** [comment]
A simple switch would probably suffice, like the one from docusaurus pages.
